src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -109,6 +110,22 @@ public function iAmLoggedInCustomer()
         $this->sharedStorage->setCurrentResource('user', $user);
     }
 
+    /**
+     * @Given I am logged in as administrator
+     */
+    public function iAmLoggedInAsAdministrator()
+    {
+        $this->prepareSessionIfNeeded();
```

Comment:
After [this](https://github.com/Sylius/Sylius/pull/4009/files#r53322959) we can remove that line.